content/uc-doc/installation/install-system.md

```diff
@@ -4,4 +4,4 @@ title: Installing the System
 
 ## The install procedure has moved
 
-Please refer to the new install documentation here : <https://wazo-platform.org/install>
\ No newline at end of file
+Please refer to the new install documentation here : <https://wazo-platform.org/install/unified-communication>
```

Comment:
it should be removed and the links pointing here should go to /install/unified-communication

Reply:
Good point, fixed!